src/styles/mixins/layout.scss

```diff
@@ -60,24 +60,34 @@
  * Establish a horizontal (margin) rhythm for this element's immediate
  * children (i.e. put equal space between children).
  *
+ * This mixin uses `!important` such that it can compete with specificity
+ * of reset rules that set some of our element's margins to 0. That allows
+ * these rules—which are applied to a parent element—to be able to assert
+ * margins, as it should be able to do.
+ *
  * @param $size [5px] - Size of horizontal margin between child elements
  */
 @mixin horizontal-rhythm($size: 5px) {
   & > * + * {
-    margin-left: $size;
+    margin-left: $size !important;
   }
 }
 
 /**
  * Establish a vertical (margin) rhythm for this element's immediate
  * children (i.e. put equal space between children).
  *
+ * This mixin uses `!important` such that it can compete with specificity
+ * of reset rules that set some of our element's margins to 0. That allows
+ * these rules—which are applied to a parent element—to be able to assert
+ * margins, as it should be able to do.
+ *
  * @param $size [var.$layout-space--medium]: Spacing size (padding)
  * @FIXME Find workaround for SvgIcon exception
  */
 @mixin vertical-rhythm($size: var.$layout-space) {
   & > * + *:not([class*='SvgIcon--inline']) {
-    margin-top: $size;
+    margin-top: $size !important;
```

Comment:
This has the following effect on the notebook modal:
![image](https://user-images.githubusercontent.com/8555781/113304428-50d0a200-9302-11eb-862a-cde4fba96aa3.png)

I am aware that the button in the modal is going to be replaced, but meanwhile this would be the effect.

Reply:
Wow! Thank you for catching this, @esanzgar ! I'll take a look ASAP.